spring-boot/src/main/java/com/sentrydemos/springboot/Application.java

```diff
@@ -41,6 +41,18 @@ public void initSentry() {
 			options.setEnvironment(environment);
 			options.setTracesSampleRate(1.0);
 			options.setRelease(getRelease());
+			
+			options.setTracesSampler(
+				context -> {
+					//context.getTransactionContext().getName() returns String: GET /products
+					if (context.getTransactionContext().getOperation().equals("http.server") &&
+							context.getTransactionContext().getName().startsWith("OPTIONS")) {
+						//Not sampling OPTIONS transactions
```

Comment:
Maybe I'm stuck on semantics but I think we are sampling here. I'd review what the comments say in /flask or /express and try to mimic those comments. @sdzhong

Reply:
We are looking for OPTIONS requests--if so, return 0.0; else, return 1.0. I am matching the approach in our Express code. Here it is for [reference](https://github.com/sentry-demos/application-monitoring/blob/932ca8fca48115bc397c61eb7b16b055aeb49776/express/server.js#L82-L86): 
```
if (request && request.method == 'OPTIONS') {
  return 0.0
}  else {
  return 1.0
}
```